libmachine/mcnutils/b2d.go

```diff
@@ -20,12 +19,8 @@ var (
 	GithubApiToken string
 )
 
-const (
-	timeout = time.Second * 5
-)
-
 func defaultTimeout(network, addr string) (net.Conn, error) {
-	return net.DialTimeout(network, addr, timeout)
+	return net.Dial(network, addr)
```

Comment:
@nathanleclaire It should be yes. I don't understand why it fails here even with a >30s timeout.